frontend/src/components/uam/common/enums/user-table-accesson.enum.ts

```diff
@@ -0,0 +1,8 @@
+enum UserTableAccesson {
```

Comment:
```suggestion
enum PermissionUserTableAccesson {
```

Reply:
also believe it should be ...Accesso**r**